### DISCUSSION TITLE
EVRS accumulated

### DISCUSSION THREAD

**Discussion by ThaReality** [EvernodeXRPL/evernode-host#106] [General]
Hi, what happens with the EVRS we've accumulated as part of our nodes testing?
I had accumulated in excess of 300,000 EVRS if not more across 3 nodes.

However, at some point there were issues with upgrading the client which resulted in a couple going dormant until a solution was found.
I kept the secret seeds and have tried to restore the nodes but keep getting address not found.

It should be noted (as you're already aware) that sashimono creates and assign the XRPL wallet itself and therefore all we could do is go into config and capture the secret seed and host address/wallet.
I checked Bithomp and I can't see any of the accumulated EVRS, is it because the account isn't activated?

![image](https://github.com/EvernodeXRPL/evernode-host/assets/75538827/74a7775a-c66b-4ecb-bba8-a594c6b9d092)

**Comment by wojake** [comment]
Could you please send me your hosts' XRPL account address?

**Reply by ThaReality** [reply]
Hi, please ignore the other user, that's not me.

As requested, these are the addresses that all run a Node

rnET2YR19WDP4vB8XtDhcF2J4afqMM6xim
rpow1rR27whTfy5JHvU7GSt7FjU2kcQHEY 
rhN9d79ZhkGSpM9D4H6nhoAAiWyPtPLMX5 

Look forward to hearing from you.

**Reply by wojake** [reply]
None of the accounts are funded on the V3 testnet, meaning, you didn't migrate from the V2 testnet to the V3 testnet for Evernode's beta test. Could you remember if you had migrated your server from V2 to V3, by any chance?

**Reply by ThaReality** [reply]
I certainly may not have done that. O well, I guess that's all lost then.

**Reply by rippleitinnz** [reply]
Back when we migrated from V2 to V3 they gave instruction regarding backing up our seeds for the V2 wallets and to store them safely, for claiming later. Perhaps you've done this and forgotten about it. It would be worth doing a search for those.

**Comment by TheMainlander** [comment]
Hi Wojak,










Mate, Kumo here!










Weird getting email from you but very cool!










Yes, I will dig all 3 up. Not my private one right 🤪










Just the 3 Prod ones on the EVR console?










Hope you're well.







Also... how do I know it's you?! 🤔







Thanks,







Kumo.






















Sent from Proton Mail mobile
















\-------- Original Message --------
On 2 Sept 2023, 6:14 pm, Wo Jake < ***@***.***> wrote:

>
>
>
> Could you please send me your hosts' XRPL account address?
>
> —
> Reply to this email directly, [view it on GitHub][], or [unsubscribe][].
> You are receiving this because you are subscribed to this thread.![AJI4NJARKMKKODIS7W5XF7LXYLTFDA5CNFSM6AAAAAA4FZCZASWGG33NNVSW45C7OR4XAZNRIRUXGY3VONZWS33OINXW23LFNZ2KUY3PNVWWK3TUL5UWJTQANEUDM.gif][]Message ID: ***@***.***>


[view it on GitHub]: https://github.com/EvernodeXRPL/evernode-host/discussions/106#discussioncomment-6891574
[unsubscribe]: https://github.com/notifications/unsubscribe-auth/AJI4NJBSQGFYGKS7HOVDNSDXYLTFDANCNFSM6AAAAAA4FZCZAQ
[AJI4NJARKMKKODIS7W5XF7LXYLTFDA5CNFSM6AAAAAA4FZCZASWGG33NNVSW45C7OR4XAZNRIRUXGY3VONZWS33OINXW23LFNZ2KUY3PNVWWK3TUL5UWJTQANEUDM.gif]: https://github.com/notifications/beacon/AJI4NJARKMKKODIS7W5XF7LXYLTFDA5CNFSM6AAAAAA4FZCZASWGG33NNVSW45C7OR4XAZNRIRUXGY3VONZWS33OINXW23LFNZ2KUY3PNVWWK3TUL5UWJTQANEUDM.gif

**Reply by wojake** [reply]
LOL, this is a GitHub comment, not a gmail btw!

**Comment by TheMainlander** [comment]
Oh shit.







😆



















Sent from Proton Mail mobile











\-------- Original Message --------
On 2 Sept 2023, 7:52 pm, Wo Jake < ***@***.***> wrote:

>
>
>
> LOL, this is a GitHub comment, not a gmail btw!
>
> —
> Reply to this email directly, [view it on GitHub][], or [unsubscribe][].
> You are receiving this because you commented.![AJI4NJD6LNRSUJYQJ7IJ2ULXYL6VJA5CNFSM6AAAAAA4FZCZASWGG33NNVSW45C7OR4XAZNRIRUXGY3VONZWS33OINXW23LFNZ2KUY3PNVWWK3TUL5UWJTQANEUVS.gif][]Message ID: ***@***.***>


[view it on GitHub]: https://github.com/EvernodeXRPL/evernode-host/discussions/106#discussioncomment-6891865
[unsubscribe]: https://github.com/notifications/unsubscribe-auth/AJI4NJD6JQYXN5NTBT3OSGLXYL6VJANCNFSM6AAAAAA4FZCZAQ
[AJI4NJD6LNRSUJYQJ7IJ2ULXYL6VJA5CNFSM6AAAAAA4FZCZASWGG33NNVSW45C7OR4XAZNRIRUXGY3VONZWS33OINXW23LFNZ2KUY3PNVWWK3TUL5UWJTQANEUVS.gif]: https://github.com/notifications/beacon/AJI4NJD6LNRSUJYQJ7IJ2ULXYL6VJA5CNFSM6AAAAAA4FZCZASWGG33NNVSW45C7OR4XAZNRIRUXGY3VONZWS33OINXW23LFNZ2KUY3PNVWWK3TUL5UWJTQANEUVS.gif

**Comment by TheMainlander** [comment]
Oh thank god.







I've sent 100 Million xrp to each addy.







Let me know when this arrives.







It can take a while as each Zerp has to be stamped for authenticity by Joel.







😎🤝🏼



















Sent from Proton Mail mobile











\-------- Original Message --------
On 2 Sept 2023, 9:47 pm, ThaReality < ***@***.***> wrote:

>
>
>
> Hi, please ignore the other user, that's not me.
>
> As requested, these are the addresses that all run a Node
>
> rnET2YR19WDP4vB8XtDhcF2J4afqMM6xim
> rpow1rR27whTfy5JHvU7GSt7FjU2kcQHEY
> rhN9d79ZhkGSpM9D4H6nhoAAiWyPtPLMX5
>
> Look forward to hearing from you.
>
> —
> Reply to this email directly, [view it on GitHub][], or [unsubscribe][].
> You are receiving this because you commented.![AJI4NJCL3M4GBTS5HNL333LXYMMGRA5CNFSM6AAAAAA4FZCZASWGG33NNVSW45C7OR4XAZNRIRUXGY3VONZWS33OINXW23LFNZ2KUY3PNVWWK3TUL5UWJTQANEVPM.gif][]Message ID: ***@***.***>


[view it on GitHub]: https://github.com/EvernodeXRPL/evernode-host/discussions/106#discussioncomment-6892278
[unsubscribe]: https://github.com/notifications/unsubscribe-auth/AJI4NJBUIPYMOWKCA7PFMNTXYMMGRANCNFSM6AAAAAA4FZCZAQ
[AJI4NJCL3M4GBTS5HNL333LXYMMGRA5CNFSM6AAAAAA4FZCZASWGG33NNVSW45C7OR4XAZNRIRUXGY3VONZWS33OINXW23LFNZ2KUY3PNVWWK3TUL5UWJTQANEVPM.gif]: https://github.com/notifications/beacon/AJI4NJCL3M4GBTS5HNL333LXYMMGRA5CNFSM6AAAAAA4FZCZASWGG33NNVSW45C7OR4XAZNRIRUXGY3VONZWS33OINXW23LFNZ2KUY3PNVWWK3TUL5UWJTQANEVPM.gif
